xsd.c

```diff
@@ -567,17 +559,6 @@ maybe_invalid_time_url(term_t type)
 }
 
 
-static int
```

Comment:
I would be inclined to keep this function in there and simply change it to return `year*sign`.   The compiler will remove it anyway.  It makes the change easier to understand and would facilitate a mode to support the old XSD stuff if we would want to.

Reply:
Yes, that's a good idea.  I've now added XSD 1.0 behavior through a C preprocessor directive.  Is this indeed the best way to achieve this?

I would like to make the Prolog test file depend on this directive as well, running either XSD 1.0 or 1.1 tests, based on the version for which `xsd.c` was compiled.  Is there a way to communicate this preprocessor directive between the C source file and the Prolog test file?